posix.mak

```diff
@@ -522,8 +521,8 @@ checkwhitespace: $(LIB) $(TOOLS_DIR)/checkwhitespace.d
 #############################
 
 ../dscanner:
-	git clone https://github.com/Hackerpilot/Dscanner ../dscanner
-	git -C ../dscanner checkout tags/v0.4.0
+	git clone https://github.com/dlang-community/Dscanner ../dscanner
+	git -C ../dscanner checkout phobos
```

Comment:
I have enabled force-push protection and a passing CI status requirement for `phobos` branch at [dlang-community](https://github.com/dlang-community), so we should be safe from accidentally breaking the Phobos setup.